website/docs/about/contribution.md

```diff
@@ -1,34 +1,110 @@
 ---
 title: Contribution
+description: How to collaborate with the Design System team and contribute to the system.
```

Comment:
```suggestion
description: How to collaborate with the Design Systems team and contribute to the system.
```

Reply:
We should discuss what to standardize on but I thought the usual term was "design systems" with an "s".

Reply:
Ah, good point. I just used what was in the original version, but I checked our [Writing Guidelines](https://docs.google.com/document/d/1WyoJVpWFVgWbCnZ28WW0gRJwJlj-qV5oY0ExhDR2obs/edit#heading=h.bnsoklnnc6rw) and we did define a consistent approach to this. I'll update all the instances to use "Design Systems Team" to match what is likely being used most across the website. Thanks for noting!

Reply:
Yup, I specifically called that out in my last comment and it's why I haven't accepted any of the suggestions for it. 🙂  I'm working on the fixes now and will have them committed soon for re-review.

Reply:
Sorry didn't read closely enough. Thanks!

Reply:
No worries, @KristinLBradley. Thanks for the thorough review! I think I caught all of your suggestions, and it's ready for re-review. Let me know if there's anything else.